src/pybind/mgr/cephadm/utils.py

```diff
@@ -22,17 +24,18 @@ def name_to_config_section(name: str) -> str:
     else:
         return 'mon'
 
-def name_to_auth_entity(daemon_type,  # type: str
-                        daemon_id,    # type: str
-                        host = ""     # type  Optional[str] = ""
```

Comment:
the `host` param was not being type checked.

Reply:
duplicates with #36497 a bit

Reply:
> duplicates with #36497 a bit

uhm yeah ...